dls-controls/malcolmjs
# Feature/plot interaction #353

## Description

Plot now retains user pan/zoom state, and reverts to default behaviour of showing last 30 seconds of data when the home button in the plotly toolbar is clicked. Also implemented workaround for bug causing errors and bad behaviour when interacting with the plot during update.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:INENC1/val
- click on the plot tab in the middle pane
- observe the default behaviour
- click on the pan or zoom icon in the plotly toolbar and interact with the plot.
- click on the home icon in the plotly toolbar and check the plot reverts to showing the last 30 seconds of data

## Agile board tracking

connect to #353 
closes #353 
connect to #348 
closes #348
connect to #284
